app/src/main/java/com/duckduckgo/app/waitlist/WaitlistNotificationDialog.kt

```diff
@@ -49,6 +49,8 @@ class WaitlistNotificationDialog : DialogFragment() {
     }
 
     companion object {
+        const val NOTIFICATION_DIALOG_TAG = "NOTIFICATION_DIALOG_FRAGMENT"
```

Comment:
Moving from an email class that's now deleted to the dialog. Also moving the dialog to a different package because it is also used by the appTp waitlist